tests/Test.ADAL.NET.Unit/BrokerParametersTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Cache;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.ClientCreds;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Flows;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Helpers;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Instance;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Test.ADAL.NET.Common;
+
+namespace Test.ADAL.NET.Unit
+{
+    [TestClass]
+    public class BrokerParametersTests
+    {
+        private const string Authority = "https://login.microsoftonline.com/test";
+        private static readonly string CanonicalizedAuthority = Authenticator.CanonicalizeUri(Authority);
+
+        private const string ExtraQueryParameters = "testQueryParameters";
+        private const string Claims = "testClaims";
+        private const string Resource = "testResource";
+
+        private const string ClientId = "testClientId";
+        private const string ClientSecret = "testClientSecret";
+
+        private const string UniqueUserId = "testUniqueUserId";
+
+        private readonly RequestData _requestData = new RequestData
+        {
+            Authenticator = new Authenticator(Authority, false),
+            Resource = Resource,
+            ClientKey = new ClientKey(new ClientCredential(ClientId, ClientSecret)),
+            SubjectType = TokenSubjectType.Client,
+            ExtendedLifeTimeEnabled = false
+        };
+
+        [TestMethod]
+        [Description("Test setting of brokerParameters by AcquireTokenInteractiveHandler constructor")]
+        public void AcquireTokenInteractiveHandlerConstructor_InitializeBrokerParameters()
+        {
+            var acquireTokenInteractiveHandler = new AcquireTokenInteractiveHandler(_requestData,
+                TestConstants.DefaultRedirectUri, null, UserIdentifier.AnyUser,
+                ExtraQueryParameters, null, Claims);
+
+            Assert.AreEqual(11, acquireTokenInteractiveHandler.brokerParameters.Count);
+
+            var brokerParams = acquireTokenInteractiveHandler.brokerParameters;
+
+            Assert.AreEqual(CanonicalizedAuthority, brokerParams["authority"]);
```

Comment:
>"authority" [](start = 65, length = 11)

was it ever considered to add these names to a static (parameter) helper class?

Reply:
good point,  will add